CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v0.47.1
```

Comment:
Should be a minor release – Patch releases are just for bug fixes.

I've approved the other PR, so that'll likely get merged first – making this `v0.49.0`?